target/linux/ath79/dts/ar9331_onion_omega.dts

```diff
@@ -0,0 +1,141 @@
+// SPDX-License-Identifier: GPL-2.0
```

Comment:
you may consider gpl2 or later, or even dual licensed, see other files for examples, but strict gpl2 _only_ is perhaps (or not!) not whatyou intended?

Reply:
I took the license from the original ar71xx commit; as I was not sure whether work based on that could be re-licensed without the original authors consent?